src/pybind/mgr/dashboard/frontend/src/app/shared/services/api-interceptor.service.ts

```diff
@@ -22,6 +22,7 @@ export class CdHttpErrorResponse extends HttpErrorResponse {
   preventDefault: Function;
   ignoreStatusCode: Function;
 }
+export const interceptorSkipHeader = 'X-Skip-Interceptor';
```

Comment:
I would use the CD instead of X, so we know its a header we added:
This can be ignored if there is a reason to use X instead.
```suggestion
export const interceptorSkipHeader = 'Cd-Skip-Interceptor';
```